### PR TITLE
Chore/lint react tokens

### PR DESCRIPTION
**Pull request by evwilkin**
**What**: Towards #3592 - resolves existing linter errors for react-tokens

### CODE REVIEW

File: packages/patternfly-4/react-tokens/src/generateTokens.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* eslint-disable global-require,import/no-dynamic-require,no-restricted-globals */
+/* eslint-disable @typescript-eslint/no-var-requires */
```
Comment (dlabrecq) [inline packages/patternfly-4/react-tokens/src/generateTokens.js]: Should we remove the no-var-requires rule? Or perhaps have a different linter config for js?

Comment (dlabrecq) [reply packages/patternfly-4/react-tokens/src/generateTokens.js]: Isn't this more of a build script. Should we apply the linter here?

Comment (dlabrecq) [inline packages/patternfly-4/react-tokens/src/generateTokens.js]: Revert the no-var-requires rule

Comment (evwilkin) [reply packages/patternfly-4/react-tokens/src/generateTokens.js]: Remove no-var-requires rule

**Pull request merged**